azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -462,15 +465,24 @@ public void prefetchToken(final File tokenFile, final Props props, final Logger
     doPrefetch(tokenFile, props, logger, userToProxy);
   }
 
+  private String kerberosSuffix(final Props props) {
+    final String host = ServiceProvider.SERVICE_PROVIDER.getInstance(Props.class)
+        .getString(AZKABAN_SERVER_HOST_NAME, "unknown");
+    final StringBuilder builder = new StringBuilder("az_");
+    builder.append(host);
+    builder.append("_");
+    builder.append(props.getString(FlowProperties.AZKABAN_FLOW_EXEC_ID));
+    builder.append(props.getString(HadoopSecurityManager.DOMAIN_NAME));
+    return builder.toString();
+  }
   private void doPrefetch(final File tokenFile, final Props props, final Logger logger,
       final String userToProxy) throws HadoopSecurityManagerException {
     // Create suffix to be added to headless user, the suffix is a valid email ID which conforms
```

Comment:
Suffix logic has changed, can you please fix the comment to reflect the new logic?

Reply:
Will do.